src/server/plugins/engine/pageControllers/SummaryPageController.test.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  ComponentType,
+  type ComponentDef,
+  type FormDefinition
+} from '@defra/forms-model'
+import { type Request } from '@hapi/hapi'
+import { format } from 'date-fns'
+
+import { getPersonalisation } from './SummaryPageController.js'
+
+import {
+  FormModel,
+  SummaryViewModel
+} from '~/src/server/plugins/engine/models/index.js'
+
+describe('getPersonalisation', () => {
```

Comment:
Think the old description has been left in?

Or if not, let's move and rename this describe block

```suggestion
describe('SummaryPageController', () => {
  describe('getPersonalisation', () => {
```